static/app/components/core/button/index.chonk.tsx

```diff
@@ -150,7 +152,7 @@ export function getChonkButtonStyles(
             },
           },
 
-          '&:active, &[aria-expanded="true"]': {
+          '&:active, &[aria-expanded="true"], &[aria-checked="true"]': {
```

Comment:
`segmentedControl` is a checkbox and thus sets `aria-checked` for “active” elements